lib/PuppeteerSharp.Tests/BrowserContextTests/BrowserContextOverridePermissionsTests.cs

```diff
@@ -92,30 +90,31 @@ public async Task ShouldIsolatePermissionsBetweenBrowserContexts()
             var otherContext = await Browser.CreateBrowserContextAsync();
             var otherPage = await otherContext.NewPageAsync();
             await otherPage.GoToAsync(TestConstants.EmptyPage);
-            Assert.AreEqual("prompt", await GetPermissionAsync(Page, "geolocation"));
-            Assert.AreEqual("prompt", await GetPermissionAsync(otherPage, "geolocation"));
+            Assert.That(await GetPermissionAsync(Page, "geolocation"), Is.EqualTo("prompt"));
+            Assert.That(await GetPermissionAsync(otherPage, "geolocation"), Is.EqualTo("prompt"));
 
             await Context.OverridePermissionsAsync(TestConstants.EmptyPage, new OverridePermission[] { });
             await otherContext.OverridePermissionsAsync(TestConstants.EmptyPage, new OverridePermission[] { OverridePermission.Geolocation });
-            Assert.AreEqual("denied", await GetPermissionAsync(Page, "geolocation"));
-            Assert.AreEqual("granted", await GetPermissionAsync(otherPage, "geolocation"));
+            Assert.That(await GetPermissionAsync(Page, "geolocation"), Is.EqualTo("denied"));
+            Assert.That(await GetPermissionAsync(otherPage, "geolocation"), Is.EqualTo("granted"));
 
             await Context.ClearPermissionOverridesAsync();
-            Assert.AreEqual("prompt", await GetPermissionAsync(Page, "geolocation"));
-            Assert.AreEqual("granted", await GetPermissionAsync(otherPage, "geolocation"));
+            Assert.That(await GetPermissionAsync(Page, "geolocation"), Is.EqualTo("prompt"));
+            Assert.That(await GetPermissionAsync(otherPage, "geolocation"), Is.EqualTo("granted"));
 
             await otherContext.CloseAsync();
         }
 
+        [Test, Ignore("previously not marked as a test")]
         public async Task AllEnumsdAreValid()
```

Comment:
Doing my homework and checking these tests.

```suggestion
        [Test]
        public async Task AllEnumsAreValid()
```

Reply:
This test consistently fails for me in Firefox:

```
 AllEnumsdAreValid
   Source: BrowserContextOverridePermissionsTests.cs line 109
   Duration: 3,4 sec

  Message: 
PuppeteerSharp.MessageException : Protocol error (Browser.grantPermissions): Browser.grantPermissions RemoteAgentError@chrome://remote/content/cdp/Error.sys.mjs:18:5
UnknownMethodError@chrome://remote/content/cdp/Error.sys.mjs:101:7
execute@chrome://remote/content/cdp/domains/DomainCache.sys.mjs:92:13
execute@chrome://remote/content/cdp/sessions/Session.sys.mjs:59:25
onPacket@chrome://remote/content/cdp/CDPConnection.sys.mjs:257:36
onMessage@chrome://remote/content/server/WebSocketTransport.sys.mjs:127:18
handleEvent@chrome://remote/content/server/WebSocketTransport.sys.mjs:109:14


  Stack Trace: 
TaskHelper.WithTimeout[T](Task`1 task, TimeSpan timeout, Func`2 exceptionFactory) line 188
Connection.SendAsync(String method, Object args, Boolean waitForCallback, CommandOptions options) line 138
BrowserContextOverridePermissionsTests.AllEnumsdAreValid() line 112
GenericAdapter`1.BlockUntilCompleted()
NoMessagePumpStrategy.WaitForCompletion(AwaitAdapter awaiter)
AsyncToSyncAdapter.Await[TResult](Func`1 invoke)
AsyncToSyncAdapter.Await(Func`1 invoke)
TestMethodCommand.RunTestMethod(TestExecutionContext context)
TestMethodCommand.Execute(TestExecutionContext context)
<>c__DisplayClass1_0.<Execute>b__0()
DelegatingTestCommand.RunTestMethodInThreadAbortSafeZone(TestExecutionContext context, Action action)

```

Reply:
Ah. I thought it was just a local check. Let's leave this as ignored. I'll write a rule to ignore it in Firefox in a follow-up PR.